### ISSUE TITLE
StateTrackingManager error on new created event save when undoredo widget exists

### ISSUE THREAD

**Issue by chuckn0rris** [bryntum/support#4715]
[Forum post](https://www.bryntum.com/forum/viewtopic.php?f=51&t=20489&p=105985#p105985)
[Forum post 2](https://www.bryntum.com/forum/viewtopic.php?f=44&t=22019&p=108839#p108839)

Reproduced in resourceHistogram demo

Add stm config into project
```
    stm : {
        autoRecord: true
    },
```

Important! Add undoredo widget
```
    tbar : {
        items: [{
            id    : 'undoredo',
            ref   : 'undoredoTool',
            type  : 'undoredo',
            items : {transactionsCombo : null}
        }]
    }
```

Open demo, dblclick on timeaxis to open task editor. Click Save without any changes. See an error in console:
```
StateBase.js:11 Uncaught (in promise) Error: Method cannot be called at this state!
    at throwInvalidMethodCall (StateBase.js:11:11)
    at AutoReadyStateClass.onStopTransaction (ReadyState.js:109:9)
    at stateTransition (StateTrackingManager.js:86:26)
    at StateTrackingManager.stopTransaction (StateTrackingManager.js:518:9)
    at TaskEdit.commitStmTransaction (ProTaskEditStm.js:53:13)
    at TaskEdit.save (TaskEdit.js:936:16)
```

**Comment by marciogurka** [comment]
Another [Post](https://www.bryntum.com/forum/viewtopic.php?f=44&t=22019&p=108839#p108839) regarding this issue.

**Comment by canonic-epicure** [comment]
Reproduced

**Comment by canonic-epicure** [comment]
Actually this is a duplicate of #3530 (regression after #6699) - not reproducible in the `3530-stm-event-create` branch. Please track #3530 for progress.
